modules/oc-mirror-enclave-support-about.adoc

```diff
@@ -19,4 +19,35 @@ Enclave support has the following benefits:
 
 * You can mirror content directly from the centralized internal registry to enclaves without restarting the mirroring process from the internet for each enclave.
 
-* You can minimize data transfer between network stages, so to ensure that a blob or image is transferred only once from one stage to another.
\ No newline at end of file
+* You can minimize data transfer between network stages, so to ensure that a blob or image is transferred only once from one stage to another.
+
+== How to mirror to an enclave
+
+image::445_OpenShift_Enclave_support_0724.png[Enclave Support]
+
+The above illustration outlines the flow for using the oc-mirror plugin in different environments, including environments with and without an internet connection.
+
+*Environment with Internet Connection*:
+
+- The user executes oc-mirror plugin V2 to mirror content from an online registry to a local disk directory.
+- The mirrored content is saved to the disk for transfer to offline environments.
+
+*Airgapped Enterprise Environment (No Internet)*:
```

Comment:
@sherine-k and @lmzuccarelli can "airgapped" be replaced with "disconnected" here? Per our [repo's term glossary](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#disconnected), "disconnected environment" is always preferred over "restricted" and "air-gapped", and we generally try to stay consistent with our wording.

In this case, I think that because "air gapped" is already used in the image above, it's not worth changing in this PR, but if "disconnected" is at all interchangeable with "air gapped" in this context, I think this is a fix we'll want to make down the line.

Reply:
Let me change the phrase "airgapped" to "disconnected".

Reply:
Sounds good, just keep in mind that at some point, the image needs to be updated to remove "airgapped" as well, and that right now there's a sort of awkward discrepancy in the phrasing.